app/assets/stylesheets/all/form_items.scss

```diff
@@ -0,0 +1,86 @@
+@import 'mixins';
+
+.form-items #form-items-header {
+  @include action-links;
```

Comment:
I think I noticed today that the icons here weren't orange. Please make sure they are.